examples/ios/swift-2.2/GettingStarted.playground/Contents.swift

```diff
@@ -63,9 +67,15 @@ try! realm.write {
 
 let favorites = ["Jennifer"]
 
+#if swift(>=3.0)
+let favoritePeopleWithSpousesAndCars = realm.allObjects(ofType: Person.self)
+    .filter(using: "cars.@count > 1 && spouse != nil && name IN %@", favorites)
+    .sorted(onProperty: "age")
```

Comment:
Comparing this Swift 3 block with the Swift 2 API really highlights how much verbosity has been introduced. The only additions that feel like they add value, IMO, is the introduction of the `onProperty` label for `sorted()`.